masonite/providers/HelpersProvider.py

```diff
@@ -22,3 +22,9 @@ def boot(self, View, ViewClass, Request):
         builtins.resolve = self.app.resolve
 
         ViewClass.share({'request': Request.helper, 'auth': Request.user})
+
+        # Share token csrf
+        token = Request.get_cookie('csrftoken')
+        ViewClass.share(
+            "<input type='hidden' name='csrftoken' value='" + token + "' />"
```

Comment:
This needs to be inside it's own `CSRFProvider` file. This should also be:

```python
ViewClass.share('csrf_field', "<input type='hidden' name='csrf_token' value='{0}' />".format(token))
```

then inside the template, we can do:

```html

<form action="" method="">
    {{ csrf_field }}
    <input type=.....
</form>

```